test-runner/src/main/scala/scala/scalanative/testinterface/ComRunner.scala

```diff
@@ -69,7 +68,7 @@ class ComRunner(bin: File,
   def receive(timeout: Duration = Duration.Inf): Message =
     synchronized {
       in.mark(Int.MaxValue)
-      val savedSoTimeout = socket.getSoTimeout()
+      val savedSoTimeout = socket.getSoTimeout
```

Comment:
The changes in this file look like they could/should be in a different PR as well.